examples/module/mpc/cartpole.py

```diff
@@ -0,0 +1,146 @@
+import os
+import torch
+import shutil
+import argparse
+import setproctitle
+import pypose as pp
+import pickle as pkl
+import torch.optim as optim
+
+def main():
```

Comment:
I run this example. It converges really slow and cannot converge to the gt values, still significant amount of error.